src/simple_idml/indesign/scripts/export.jsx

```diff
@@ -204,24 +210,41 @@ try {
                 omitEPS = false;
                 omitPDF = false;
                 pageInformationMarks = _pageInformationMarks;
-                pageMarksOffset = _pageMarksOffset;
-                pdfMarkType = MarkTypes.DEFAULT_VALUE;
+                try{
+                    pageMarksOffset = _pageMarksOffset;
+                }
+                catch(e) {alert("Ignoring exception setting property pageMarksOffset: " + e)}
+                try{
+                    pdfMarkType = MarkTypes.DEFAULT_VALUE;
+                }
+                catch(e) { alert("ignoring exception setting property pdfMarkType: " + e)}
                 printerMarkWeight = PDFMarkWeight.p125pt;
                 registrationMarks = _registrationMarks;
                 try {
                     simulateOverprint = false;
                 }
-                catch(e){}
+                catch(e) { alert("ignoring exception setting property simulateOverprint: " + e)}
                 //Set viewPDF to true to open the PDF in Acrobat or Adobe Reader.
-                viewPDF = false;
+                //viewPDF = false;
```

Comment:
Is it wanted?

Reply:
Ah, good catch. I'll convert that to a try/catch block like the others.